### PR TITLE
Adding icons with priority

### PR DESCRIPTION
**Pull request by jairochaves2**

### CODE REVIEW

**Review by iurynogueira — APPROVED** [review]

File: projects/ion/src/lib/icon/svgs/icons.ts [hunk 1]
```diff
@@ -1,21 +1,56 @@
+import {
+  access,
+  add,
+  check,
+  checkSolid,
+  clockSolid,
+  close,
+  closeSolid,
+  down2,
```
Comment (Deeved) [inline projects/ion/src/lib/icon/svgs/icons.ts:9]: could be doubleDown

File: projects/ion/src/lib/icon/svgs/icons.ts [hunk 2]
```diff
@@ -1,21 +1,56 @@
...
+  exclamationSolid,
+  expand,
+  filter,
+  infoSolid,
+  left2,
```
Comment (Deeved) [inline projects/ion/src/lib/icon/svgs/icons.ts:14]: could be doubleLeft

File: projects/ion/src/lib/icon/svgs/icons.ts [hunk 3]
```diff
@@ -1,21 +1,56 @@
...
+  option,
+  pencil,
+  plusSolid,
+  refresh,
+  right2,
```
Comment (Deeved) [inline projects/ion/src/lib/icon/svgs/icons.ts:19]: could be doubleRight

File: projects/ion/src/lib/icon/svgs/icons.ts [hunk 4]
```diff
@@ -1,21 +1,56 @@
...
+  search,
+  semiDown,
+  semiUp,
+  starSolid,
+  trash,
+  union,
+  up2,
```
Comment (Deeved) [inline projects/ion/src/lib/icon/svgs/icons.ts:26]: could be doubleUp

File: projects/ion/src/lib/icon/svgs/icons.ts [hunk 5]
```diff
@@ -1,21 +1,56 @@
+import {
```
Comment (Deeved) [inline projects/ion/src/lib/icon/svgs/icons.ts:1]: very good! could you create the calendar icon?

File: projects/ion/src/lib/icon/svgs/icons.ts (hunk 1 above)
Comment (jairochaves2) [reply projects/ion/src/lib/icon/svgs/icons.ts:9]: I'm following figma nomenclature. Wouldn't modifying cause any problems?

Comment (jairochaves2) [reply projects/ion/src/lib/icon/svgs/icons.ts:9]: image

Comment (jairochaves2) [reply projects/ion/src/lib/icon/svgs/icons.ts:9]: image

File: projects/ion/src/lib/icon/svgs/icons.ts (hunk 5 above)
Comment (jairochaves2) [reply projects/ion/src/lib/icon/svgs/icons.ts:1]: Yes.

Comment (jairochaves2) [reply projects/ion/src/lib/icon/svgs/icons.ts:1]: But in the future there will be a feature to add the others.

Comment (jairochaves2) [reply projects/ion/src/lib/icon/svgs/icons.ts:1]: Done.

**Review by Deeved — APPROVED** [review]

**Comment by MarcosOliveir4** [comment]
could you add these three icons ?

image
image
image

**Comment by jairochaves2** [comment]
@MarcosOliveir4, added.

File: projects/ion/src/lib/icon/svgs/icons.ts (hunk 1 above)
Comment (mizek1) [reply projects/ion/src/lib/icon/svgs/icons.ts:10]: Hi @jairochaves2! I don't think left2, down2, etc., are good names. I think they could be more suggestive. We could talk about it with Edney, I don't think he intended to leave the names like that in Figma.

**Review by MarcosOliveir4 — APPROVED** [review]

File: projects/ion/src/lib/icon/svgs/icons.ts (hunk 1 above)
Comment (jairochaves2) [reply projects/ion/src/lib/icon/svgs/icons.ts]: Hi, @mizek1. I removed the down2 and up2, and kept the semi-down and semi-up.

Comment (jairochaves2) [reply projects/ion/src/lib/icon/svgs/icons.ts]: Was necessary to maintain compatibility.

**Review by mizek1 — APPROVED** [review]

**Pull request merged**